### PR TITLE
addpatch: virtiofsd

### PR DESCRIPTION
**Pull request by Avimitin**
The virtiofsd package uses an unimplemented syscall `renameat` in riscv64
which causes build failure. This commit adds a compile condition to
exclude the target arch `riscv64` from building the binding to the
unsupported syscall. This commit also removes the `--target` option when
fetching source code to fix the target triple issue.

The additional compile option has been submitted to the upstream. The MR
status can be tracked at:
https://gitlab.com/virtio-fs/virtiofsd/-/merge_requests/156.

### CODE REVIEW

File: virtiofsd/riscv64.patch [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
++++ PKGBUILD
+@@ -11,16 +11,21 @@ depends=(gcc-libs glibc libcap-ng libseccomp)
+ makedepends=(cargo)
+ provides=(virtiofsd-provider)
+ conflicts=(virtiofsd-provider)
+-source=($url/-/archive/v$pkgver/$pkgname-v$pkgver.tar.gz)
+-sha512sums=('a50b3fac6d5eafc691ef4fc8808acb4dc03261f01623f6675a234d457231f96181e9cf6c8ac68b4fe1b1c97257faf56bc6d0c2776142c9dbfe82b4e9bde0e34a')
+-b2sums=('52c86ab9d4842ecd496aed3f09a98be96d7ee0eda43a2519eb066be1f648963b155386f9eac1a1c878c12fd05d928f88700c6a7a840c0f34dfa6f86caab15fec')
++source=($url/-/archive/v$pkgver/$pkgname-v$pkgver.tar.gz
++	remove-unsupport-syscall.patch)
```
Comment (Xeonacid) [inline virtiofsd/riscv64.patch:11]:
```suggestion
+	remove-unsupport-syscall.patch::https://gitlab.com/virtio-fs/virtiofsd/-/merge_requests/156.patch)
```
Better to use link to patch.

Comment (Avimitin) [reply virtiofsd/riscv64.patch]: Fixed.

Comment (Xeonacid) [reply virtiofsd/riscv64.patch]: You forgot to update checksum. :)

Comment (Avimitin) [reply virtiofsd/riscv64.patch]: Oops..It got fixed now.

**Pull request merged**